src/v/config/configuration.cc

```diff
@@ -1461,6 +1461,25 @@ configuration::configuration()
       "performance",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , cloud_storage_enable_scrubbing(
+      *this,
+      "cloud_storage_enable_scrubbing",
+      "Enable scrubbing of cloud storage partitions. The scrubber validates "
+      "the integrity of data and metadata uploaded to cloud storage",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      true)
```

Comment:
Should this be disabled for now until it's stabilized?

Reply:
Scrubbing will go out in v23.3, so we have plenty of time to get it stable. Having it enabled surfaced some bugs in the test run.